scripts/exregional_run_analysis.sh

```diff
@@ -377,6 +380,8 @@ if [ ${regional_ensemble_option:-1} -eq 5 ]  && [ ${BKTYPE} != 1  ]; then
   grid_ratio_ens="1"
   ens_fast_read=.true.
 else    
+  weight_ens_gfs=1.0
+  weight_ens_fv3sar=1.0
```

Comment:
Should weight_ens_fv3sar be 0.0 and weight_ens_gfs be 1.0?

Reply:
I suggest removing these two lines as they don't mean anything here but only cause confusion.

I checked the GSI code and it does apply the correct settings:

```
if(l_both_fv3sar_gfs_ens) then
....
else
    if (regional_ensemble_option==5) then
       n_ens_gfs=0
       n_ens_fv3sar=n_ens
    elseif (regional_ensemble_option==1) then
       n_ens_gfs=n_ens
       n_ens_fv3sar=0
....
```

Reply:
In the case of "l_both_fv3sar_gfs_ens=false" and "regional_ensemble_option=5", the subroutine using "weight_ens_gfs" is not called. On the other hand, in the case of "l_both_fv3sar_gfs_ens=false" and "regional_ensemble_option=1", the subroutine using "weight_ens_fv3sar" is not called. So, these weights can be less than one only in making ensemble B with both global and regional ensembles.

Reply:
@shoyokota I understood what you mean and I think we are talking about the same thing. 

When `l_both_fv3sar_gfs_ens = '.false.'`, the actual effect for `regional_ensemble_option=5` is ` weight_ens_gfs=0;  weight_ens_fv3sar =1` . This was and should be implemented through the GSI code itself, not by setting `weight_ens_gfs` and `weight_ens_fv3sar` in the workflow (even if you set them to some unreasonable value, GSI will modify it).  This process should be transparent to the workflow. That's why I suggest removing these two lines to avoid the extra mental workload required to understand this under the context.

Reply:
I understand. Thank you for the suggestion. I removed this part.

Reply:
Thanks!